### DISCUSSION TITLE
Monterey last update doesn't allow me to control brightness limits

### DISCUSSION THREAD

**Discussion by luco** [MonitorControl/MonitorControl#1291] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

![image](https://user-images.githubusercontent.com/246373/205693107-837ca422-e122-4b2b-b21e-1c8804f98324.png)

I can't set DDC min/max. On the previous version of Monterey I was able to set it.

Right now I'm stuck in 50% bright. in Cinema Display

### Steps to reproduce

Described below.

### Expected behavior

Able to set DDC min and max.

### Environment Information (please complete the following information)

```markdown
- macOS version:12.6.1 (
- Mac model: Mac Mini 2018
- MonitorControl version: 4.1.0
- Monitor(s): Apple Cinema Display
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
Hi there, it seems like the display is now detected as supported Apple display (which should be good news if brightness control is working). This should mean that Apple re-added native control for this display (so DDC is not being used).
